### DISCUSSION TITLE
Speed issue

### DISCUSSION THREAD

**Discussion by Whiskey68** [flybywiresim/aircraft#2394] [Help]
Hello. Im having some issue controlling speed on climb. I always find im above 250 below 10000. I just tried setting the speed manully using the speed mode but it would only stay on thrlt clmb mode putting it above 250 agian. Clearly im doing something wrong, Suggestions?

To clarify when i attempt to control the speed using the speed control. I set it to 240 then press down on the speed nob

**Comment by hotshotp** [comment]
you sure you have autothrottle on the CLB detent and not MCT detent? also make sure athr is on

**Reply by Whiskey68** [reply]
Im pretty positive it is in CLB detent. Athr is def on. I will have to try again. Second or thrid time im having the issue

**Reply by CptJohn34** [reply]
I had this problem until my son -  A320 pilot - had a look at my screen and told me that I needed to adjust the quadrant because he saw that LVR CLB was flashing in the top left corner of the ID. I'm using an Airbus Thrustmaster quadrant and side stick. The adjustment to the quadrant was made using the EFB. There are several videos showing how this is done. Try searching for "flypad Throttle Calibration".

**Comment by Whiskey68** [comment]
This is during the climb. In the CLB detent, athr is on and above 250
![20201213095150_1](https://user-images.githubusercontent.com/75712934/102015312-da933a00-3d28-11eb-9262-5810ac0db890.jpg)

**Reply by Whiskey68** [reply]
This one is with autopilot enabled. Not sure if it helps im using the dev version and I do C/D startups for every flight 
 
![20201213095427_1](https://user-images.githubusercontent.com/75712934/102015363-3cec3a80-3d29-11eb-9a73-95041f38ec23.jpg)

**Reply by mga010** [reply]
In the first picture, you seem to force a descent. That makes it impossible for the plane to hold the speed. 

The second picture is strange. It seems you are in a very steep climb with CLB thrust at 60 tons in open climb. The plane will pitch up to hold the set speed of 250 knots. But it is already at 15°. Did that situation last for a longer time?

**Reply by Whiskey68** [reply]
I didnt even notice that in the first picture. In the second, it lasted until I took control of the aircraft.

**Reply by mga010** [reply]
Now I noticed the icing message.

**Comment by Patrick3313** [comment]
Hello Whiskey,

May I suggest you provide the configation of the A/C ( TOW;TOWCG; FOB;) , and you try to reproduce the event...Maybe we can see something 

Regards,

Patrick

**Comment by Patrick3313** [comment]
Sorry, 

And WE try to reproduce the event....

**Comment by marcsoler** [comment]
Not sure if this helps but have you checked the performance data (climb phase) on your MCDU/FMGS? 

![image](https://user-images.githubusercontent.com/1502124/102492909-a6927e80-4072-11eb-8b28-72873b8aa97e.png)
